README.md

```diff
@@ -21,50 +21,84 @@
 </p>
 
 # Kitura-CORS
-Kitura CORS middleware
-
-## Summary
-Kitura [CORS](https://www.w3.org/TR/cors/) middleware.
+Kitura cross-origin resource sharing ([CORS](https://www.w3.org/TR/cors/)) middleware. A request for a resource (such as an image) outside of your origin (e.g. server) is known as a cross-origin request. CORS allows you to bypass the same-origin restriction, so that resources can be shared between different origins (i.e. servers).
```

Comment:
How about this?
> Kitura-CORS is a cross-origin resource sharing ([CORS](https://www.w3.org/TR/cors/)) middleware.  A webpage or script provided by a service may include references to resources or services hosted by another domain - such as a public service or resource.  A browser's request for such a resource is an example of a cross-origin request, and is typically prohibited under the [same-origin security policy](https://en.wikipedia.org/wiki/Same-origin_policy).
>
>CORS allows you to bypass the same-origin restriction in a controlled manner, so that resources can be shared between different domains.  A CORS-capable client makes a 'pre-flight' request to the service hosting the resource at domain B with an `Origin` header specifying domain A.  That service can then respond to the client with an `Access-Control-Allow-Origin` header which indicates whether access to this resource should be permitted.
>
>You can use CORS middleware in your Kitura application to authorize external services to access resources or APIs that you provide.